online_store/gt_sport_v1.html

```diff
@@ -0,0 +1,120 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2> <p>
```

Comment:
why are you opening a paragraph tag at the end of this line?

Reply:
True, a mistake, got rid of it.